modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/continuous/CacheContinuousQueryHandler.java

```diff
@@ -144,7 +144,7 @@
     private transient boolean skipPrimaryCheck;
 
     /** */
-    private boolean locCache;
+    private transient boolean locOnly;
```

Comment:
Are you sure that it should be transient?

Reply:
`transient` modifier has only semantic meaning here, since this class implements `Externalizable`.
All fields in this class, that are not marked as `transient`, are used in `writeExternal()` and `readExternal()` methods. The same rule should apply for this field as well.